### DISCUSSION TITLE
Proposed notebooks

### DISCUSSION THREAD

**Discussion by ktyle** [DAES433533/donder-cookbook#12] [General]
Hi team members, per [HW3 item 2](https://docs.google.com/document/d/1AIjpOHiZngswVSMoI9oybFkrAlQAff3u-JzBeBpQk7Q/edit), please respond to this discussion thread with what your individual notebook will focus on.

**Comment by KaylaLewis23** [comment]
As we discussed over what's app, I plan to do a notebook (Notebook 2) How to improve basic time series and will maybe add an interactive time scroller. Like Adam just mentioned in the whats app I think a good title for out project would be "Visualizing Data with EPA's Air Quality System (AQS) API"

**Comment by ktyle** [comment]
Enabling interactivity, such as with a time slider, is a great idea. For that, you can leverage the Holoviz tools. An example of a notebook that provides similar functionality is https://www.atmos.albany.edu/facstaff/ktyle/atm533/core/week10/03_Geoviews_ERA5.html .

**Comment by zee790** [comment]
I am planning on doing notebook 3 which includes how to plot the data over a map and maybe add wind velocities.

**Comment by wxsamdes** [comment]
I am going to create notebook 4 which goes into how to create an interactive map with air quality data.

**Comment by AMDeitsch** [comment]
I will be starting us off with Notebook 1, accessing the data from AQS by using the pyaqsapi package, then exploring the data, preparing it, and plotting it as a timeseries.
